### PR TITLE
Adding badge instructions

### LINKED ISSUE #36: Scikit-HEP badge for project READMEs
As first [brought up in the Scikit-HEP Gitter](https://gitter.im/Scikit-HEP/community?at=5e1f4a3c84894f56b6655f83) it would be nice if we designed a Scikit-HEP badge that would link to the [project webpage](https://scikit-hep.org/) that all projects would have in their `READMEs`. As the `README`s end up getting put up on PyPI during publishing it could be a useful way to get a curious user from PyPI to the project quickly. So maybe something along the lines of

[Scikit-HEP](https://scikit-hep.org/)

but something more interesting.

As @henryiii did a great job with the color pallet choices for the PyPI badges

[PyPI](https://pypi.org/project/boost-histogram/)

I'm hoping that he has some thoughts on the badge design as well. :slightly_smiling_face:

### PR DESCRIPTION
**Pull request by henryiii**
Closes #36.

### CODE REVIEW

File: pages/developers.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+nav_order: 10
+---
+
+Developer Information
+=====================
+
+Badges: Scikit-HEP
+------------------
+
+In your readme, you should have a scikit-hep project badge:
```
Comment (eduardo-rodrigues) [inline pages/developers.md]:
```suggestion
In your README, you should have a Scikit-HEP project badge:
```

File: pages/developers.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+Or an affiliated package badge if you are an affiliated package:
```
Comment (eduardo-rodrigues) [inline pages/developers.md]: package or project

Comment (henryiii) [reply pages/developers.md]: I don't think it's too important here. I think of things that have README's and PyPI/Conda-forge releases as packages. Scikit-HEP is a project.

Comment (eduardo-rodrigues) [reply pages/developers.md]: Sure

**Comment by henryiii** [comment]
See https://github.com/GooFit/GooFit. :)

**Comment by eduardo-rodrigues** [comment]
@henryiii, under https://scikit-hep.org/developer the MD badges are in the wrong order, meaning the badge info for affiliated packages in given for project packages, and vice-versa. Can you fix that?

**Comment by henryiii** [comment]
Should have been fixed about 2 minutes ago.

**Comment by eduardo-rodrigues** [comment]
And there is also a typo in "Scikit-HEP affilated" ...

**Comment by henryiii** [comment]
That one I hadn't fixed yet, thanks!

**Pull request merged**